test/RandomTest.cpp

```diff
@@ -0,0 +1,46 @@
+//
+// Created by pua66 on 3/8/2021.
+//
+
+#include "Helpers/NumberHelpers.h"
+#include "Core/Random.h"
+#include <iostream>
+#include "gtest/gtest.h"
+
+TEST(RandomTest, GoodSeedDifferentValue){
+    auto v1 = NumberHelpers::good_seed(1, 2, 3);
+    auto v2 = NumberHelpers::good_seed(1, 2, 4);
+    ASSERT_NE(v1, v2);
+}
+
+TEST(RandomTest, ShuffleStdToVector){
```

Comment:
Something to consider when writing unit tests for random number generators is that in addition to testing to make sure something random happens (e.g., shuffling an array) you also need to test to test to make sure that the same seed produces the same results. So for the shuffle operation, in addition to checking to that the shuffled list does not equal the original, you should also be checking the shuffled list against a known standard of what it should be for the sake of completeness.

An additional complication is that, for some shuffle algorithms at least, it is possible for the list input to be output as the shuffled list returned. So for a small list of numbers there is a non-zero chance that the test could fail (for large teams where unit tests are run automatically on check-in or build this is a bigger concern), so picking a list of 100 items (odds 1 in 100!) is a good call.